js/preload/preloadMain.js

```diff
@@ -110,6 +111,13 @@ function createAPI() {
         ScreenSnippet: remote.require('./screenSnippet/index.js').ScreenSnippet,
 
         /**
+         * Provides API to crash the renderer process that calls this function
+         */
+        crashRendererProcess: function () {
```

Comment:
is there away to only allow this to be run for dev builds?

Reply:
we can try with `process.env.ELECTRON_DEV` and see if that works on preload.